examples/pcap_to_parquet.py

```diff
@@ -20,32 +20,51 @@ def ip_to_integer(s):
     return struct.unpack("!I", socket.inet_aton(s))[0]
 
 
+def get_ip_protocol(s):
+    if "tcp" in s:
+        return "tcp"
+    if "UDP" in s:
+        return "udp"
+    if "EIGRP" in s:
+        return "eigrp"
+    if "ICMP" in s:
+        return "icmp"
+    return None
+
```

Comment:
My quick-and-dirty parsing approach was to split naively on spaces. Sometimes there are extra characters or punctuation in the original line, so a generic full-string search was simpler and more robust.